pywbem/_recorder.py

```diff
@@ -523,35 +522,44 @@ class LogOperationRecorder(BaseOperationRecorder):
 
     All logging calls are at the debug level.
     """
-    def __init__(self, max_log_entry_size=None):
+    def __init__(self, conn_id=None, entry_size_limit=None):
         """
         Parameters:
 
-          max_log_entry_size (:term:`integer`):
+          conn_id ():)
+            If None, string that represents an id for a particular connection
+            that is used to build the name of each logger. if conn_id is not
+            None the logging requests are identified with the logger name
+            (ex pywbem.ops) and the suffix so (ile pywbem.ops.1-2343) so that
+            any different logger names may be established for specific
+            WBEMConnections.
```

Comment:
Use the term we will introduce for the WBEMConnection.conn_id property (e.g. "connection id")

Reply:
Cleaned up.